src/pages/events/rule-events.md

```diff
@@ -0,0 +1,99 @@
+   | `equal`       | Checks whether the payload value matches the specified value. For Boolean data types, use `1` to compare to `true` and `0` to compare to `false`. |
+   | `regex`       | A regular expression that checks for matches. The specified value must be compatible with the  [regular expression match](https://www.php.net/manual/en/function.preg-match.php) |
+   | `in`          | Checks whether the payload value is one of multiple specified values. The value must be a comma-separated list. You do not need to provide additional escape characters. |
+
+*  The value to compare against. When you assign the `regex` operator, you must delimit the regular expression value with valid characters, such as forward slashes (/). For example, `/^TV .*/i`, which checks whether the string contains the string `TV`, ignoring the case of the letters.
+
+You can create rule events within your module's `io_events.xml` file or from the command line.
+
+## Define rule events in `io_events.xml`
+
+The following example creates and registers a rule event named `plugin.magento.catalog.model.resource_model.product.save_low_stock_event`. Its parent is `plugin.magento.catalog.model.resource_model.product.save`. It defines rules that trigger when all of the conditions are true:
+
+*  The value of `qty` is less than 20
+*  The value of `price` is greater than 100
+*  The `category_id` is either 3, 4, or 5
+*  The product `name` contains `TV`
+
+These fields are present and declared in the parent event.
+
+```xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="...">
+    <event name="plugin.magento.catalog.model.resource_model.product.save_low_stock_event"
+           parent="plugin.magento.catalog.model.resource_model.product.save"
+    >
+        <fields>
+            <field name="qty"/>
+            <field name="category_id"/>
+            <field name="name">
+        </fields>
+        <rules>
+            <rule>
+                <field>qty</field>
+                <operator>lessThan</operator>
+                <value>20</value>
+            </rule>
+            <rule>
+                <field>category_id</field>
+                <operator>in</operator>
+                <value>3,4,5</value>
+            </rule>
+            <rule>
+                <field>name</field>
+                <operator>regex</operator>
+                <value>/^TV .*/i</value>
+            </rule>
+        </rules>
+    </event>
+</config>
+```
+ 
+## Command line
+
+The `bin/magento events:subscribe <event_code> --force --fields=<name1> --fields=<name2>` command creates and registers custom and native Commerce events. When you also specify the `--parent <event_code>` and `--rules=<field-name>|<operator>|<value>` options, you create and register a rule event.
+
+The following command creates and registers the same rule event shown in the `io_events.xml` example:
```

Comment:
I'm wondering if it would be useful to mention here that the command will create the details for the subscription in `config.php`

It could also be added that `bin/magento events:list -v` can be used to show details for rule events